catalog/views.py

```diff
@@ -1,3 +1,34 @@
 from django.shortcuts import render
 
 # Create your views here.
+from .models import Book, Author, BookInstance, Genre
+
+
+def index(request):
+    """View function for home page of site."""
+
+    # Generate counts of some of the main objects
+    num_books = Book.objects.all().count()
```

Comment:
Chỗ này ko cần call all() nữa vì có count rồi e nhé, lược bỏ đi được
```suggestion
    num_books = Book.objects.count()
```